src/Bicep.Cli/Services/TemplateEvaluation.cs

```diff
@@ -18,15 +18,15 @@
 
 namespace Bicep.Cli.Services
 {
-    public class TemplateEvaluatorService
+    public class TemplateEvaluation
```

Comment:
I felt like `TemplateEvaluatorService` was a more intuitive name - especially now that we have a `Evaluation` class. If you want to shorten the name, then what about calling it `TemplateEvaluator`?